apps/api/src/services/queue-worker.ts

```diff
@@ -341,11 +347,20 @@ async function processJob(job: Job, token: string) {
           error: message /* etc... */,
           docs: fullDocs,
         };
-
-        await callWebhook(job.data.team_id, job.data.crawl_id, data, job.data.webhook, job.data.v1);
+          // v0 web hooks, call when done with all the data
+        if (!job.data.v1) {
+          callWebhook(job.data.team_id, job.data.crawl_id, data, job.data.webhook, job.data.v1, "crawl.completed");
+        }
+        // v1 web hooks, call when done with no data, but with event completed
+        if (job.data.v1 && job.data.webhook) {
+          callWebhook(job.data.team_id, job.data.crawl_id, [], job.data.webhook, job.data.v1, "crawl.completed");
+        }
```

Comment:
do this before gathering all data to spare server RAM

Reply:
fixed